client/src/modules/purchases/list/list.html

```diff
@@ -1,40 +1,92 @@
-<!-- breadcrumb header -->
-<bh-breadcrumb
-  path    ="PurchaseListCtrl.bcPaths"
-  button  ="PurchaseListCtrl.bcButtons"
-  print   ="PurchaseListCtrl.bcPrint">
-</bh-breadcrumb>
-
-<!-- util bar  -->
-<div class="flex-util">
-  <bh-filters-applied
-    style="max-width:90%"
-    filters="PurchaseListCtrl.formatedFilters"
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.PURCHASE</li>
+      <li class="title" translate>TREE.PURCHASE_REGISTRY</li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <div uib-dropdown dropdown-append-to-body data-action="open-tools">
+          <a class="btn btn-default" uib-dropdown-toggle>
+            <span class="fa fa-bars"></span> <span class="hidden-xs" translate>FORM.LABELS.MENU</span> <span class="caret"></span>
+          </a>
+          
+          <ul uib-dropdown-menu role="menu" class="dropdown-menu-right">
+            <li role="menuitem">
+              <a href data-method="configure" ng-click="PurchaseListCtrl.openColumnConfiguration()">
+                <i class="fa fa-columns"></i> <span translate>FORM.LABELS.COLUMNS</span>
+              </a>
+            </li> 
+
+            <li role="separator" class="divider"></li>  
+            <li role="menuitem">
+              <a href data-method="save-state" ng-click="PurchaseListCtrl.saveGridState()">
+                <i class="fa fa-save"></i> <span translate>FORM.BUTTONS.SAVE_GRID_CONFIGURATION</span>
+              </a>
+            </li>
+
+            <li role="menuitem">
+              <a href data-method="clear-state" ng-click="PurchaseListCtrl.clearGridState()">
+                <i class="fa fa-close"></i> <span translate>FORM.BUTTONS.CLEAR_GRID_CONFIGURATION</span>
+              </a>
+            </li>
+
+            <li role="separator" class="divider"></li>
+            <li role="menuitem">
+              <a ng-href="/reports/finance/purchases/?{{ PurchaseListCtrl.download('pdf') }}" download="{{ 'PURCHASES.ORDER' | translate }}">
+                <span class="fa fa-file-pdf-o"></span> <span translate>DOWNLOADS.PDF</span>
+              </a>
+
+            </li>
+            <li role="menuitem">
+              <a ng-href="/reports/finance/purchases/?{{ PurchaseListCtrl.download('csv') }}" download="{{ 'PURCHASES.ORDER' | translate }}">
+                <span class="fa fa-file-excel-o"></span> <span translate>DOWNLOADS.CSV</span>
+              </a>
+            </li>
+                               
+          </ul>
+
+        </div>
+      </div>
+        
+      <div class="toolbar-item">
+        <button
+          ng-click="PurchaseListCtrl.search()"
+          data-method="search"
+          class="btn btn-default">
+          <span class="fa fa-search"></span> <span translate>FORM.BUTTONS.SEARCH</span>
```

Comment:
If you put `class="hidden-xs"` on the FORM.BUTTONS.SEARCH element, it will be hidden on small screens.  That makes a much nicer user experience :+1: